.istanbul.yml

```diff
@@ -1,6 +1,9 @@
 instrumentation:
   include-all-sources: true
-  excludes: ['**/test/**']
+  excludes: [
+    '**/test/**,
+    src/models/index.js'
```

Comment:
Why exclude model/index.js?

I am not even sure what the purpose is of the mode/index.js. It merely exports the DB as the default object. I am not sure I agree with that.

What do you think?

Reply:
It is disabled because even though it's covered by a test:
![image](https://user-images.githubusercontent.com/20338055/55472095-eb168600-5613-11e9-8ff4-36edfa57edc3.png)
coverage fails!?
`ERROR: Coverage for statements (50%) does not meet per-file (src/models/index.js)  threshold (90%)`

Reply:
But the DB should not even be exposed by the Model.

Reply:
We definitely need to exclude this file from the coverage-check until we remove `src/models/index.js` or we find out why even though it shows the only present line is called 1x times, coverage-check fails.

Reply:
As discussed we will create a new story to fix this.